charts/victoria-metrics-anomaly/templates/configmap.yaml

```diff
@@ -1,3 +1,10 @@
+{{- if not .Values.config.reader.tenant_id -}}
+{{- $ := unset .Values.config.reader "tenant_id" }}
+{{- end -}}
+{{- if not .Values.config.writer.tenant_id -}}
+{{- $ := unset .Values.config.writer "tenant_id" }}
+{{- end -}}
+---
```

Comment:
can you please also add the same for `monitoring.push` as we also have `tenant_id` there ?

Reply:
It is not included as a part of default values so unless user sets it manually that won't be a problem for vmanomaly.

Reply:
oh, got it, thanks for clarification!